hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -130,6 +131,23 @@
 @SuppressWarnings({"checkstyle:MethodLength", "checkstyle:NPathComplexity"})
 public class TcpClientConnectionManager implements ClientConnectionManager, MembershipListener {
 
+    /**
+     * A private property to let users control the reconnection behavior of the client.
+     * <p>
+     * When enabled, the client will skip trying to connect the member list during
+     * reconnection attempts.
```

Comment:
```suggestion
     * connection and reconnection attempts.
```

Reply:
What do you mean here? While the client is connecting the cluster, there is no member list, so it wasn't checking that

Reply:
it ends up in the same situation. but, ok, we can leave as it it.